manage/CoxTrainTestManager.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+from models.CoxModel import CoxModel
+from sklearn.model_selection import KFold
+
+
+class CoxTrainTestManager:
+    """
+    Train-test manager for Cox Model.
+    """
+    def __init__(self) -> None:
+        """
+        CoxTrainTestManager class constructor.
+
+        ### Parameters :
+        None
+
+        ### Returns :
+        None
+        """
+        self.model = None
+
+    def leave_one_out_cv(self,
+                         X: np.ndarray[np.ndarray[float]],
```

Comment:
c'est pas recommandé d'utiliser ce genre de noms de variables: https://peps.python.org/pep-0008/#names-to-avoid